experiment/logviewer/gcsreader/parsing_test.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestMatchSingleLine(t *testing.T) {
+	regex := regexp.MustCompile("\"auditID\":\"39aec93e-031b-4002-8c0a-4ddcd92e250b\"")
+	lines, err := processLines(strings.NewReader(testData), regex)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(lines) != 1 {
+		t.Fatalf("Expected 1 line, got %v", len(lines))
+	}
+}
+
+func TestParseSingleLine(t *testing.T) {
+	lineStr := testData[:983]
```

Comment:
Please get rid of 983 constant.
I think you can use separate constant for this.

e.g.

const line1 = `sth`
const line2 = `sth2`
const line3 = `sth3`

and then use line1 in TestParseSingleLine and line1+line2+line3 in TestMatchSingleLine